lib/io/src/serialization/proto/schema/catrobat_image.proto

```diff
@@ -8,5 +8,5 @@ message SerializableCatrobatImage {
   uint32 width = 3;
   uint32 height = 4;
   repeated google.protobuf.Any commands = 5;
-  optional bytes backgroundImage = 6;
```

Comment:
remark: in protoc3 all fields are optional by default
this led to an error in the compiler, so it was removed

Reply:
👍